enable rtl css (bug 39288)

jdlrobson opened the pull request:
for time being i manually generated app-rtl.css using
http://cssjanus.commoner.com/
we should automate this

jrobson committed:
enable rtl css (bug 39288)

for time being i manually generated app-rtl.css using
http://cssjanus.commoner.com/
we should automate this

jrobson committed:
flip the back button (bug 39858)

re-run through css janus

awjrichards-zz commented:
According to Jon, there are still issues with these changes. Please do not merge until they are addressed.

jrobson committed:
dynamically add styles via style tag to avoid screen flash

port several definitions to common.css

jrobson committed:
init map only after styles have loaded

the map needs to be styled correctly when initialised to have the
right dimensions

jrobson committed:
Merge branch 'master' of github.com:wikimedia/WLMMobile into rtl

Conflicts:
	assets/www/js/app.js

jdlrobson commented:
These should be addressed now

bvibber commented:
Looks good testing with Arabic locale & Israel items on my 4.0.3 emulator. (No Hebrew locale configured, bah!)
